distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -0,0 +1,367 @@
+
+Execute the following command to display the status of all the ${branding} bundles:
+
+.View Status
+----
+${branding-lowercase}${at-symbol}local>list  grep -i ${branding-lowercase}
+
+----
+
+[WARNING]
+====
+Entries in the *Resolved* state are expected, they are OSGi bundle fragments.
+Bundle fragments are distinguished from other bundles in the command line console list by a field named `Hosts`, followed by a bundle number.
+Bundle fragments remain in the *Resolved* state and can never move to the *Active* state.
+====
+
+.Example: Bundle Fragment in the Command Line Console
+----
+215 | Resolved |  80 | ${project.version}     | DDF :: Platform :: Security :: Session, Hosts: 90
+----
+
+After successfully completing these steps, the ${branding} is ready to be configured.
+
+==== ${branding} Directory Contents after Installation and Initial Startup
+
+During ${branding} installation, the major directories and files shown in the table below are created, modified, or replaced in the destination directory.
+
+.${branding} Directory Contents
+[cols="1,4" options="header"]
+|===
+
+|Directory Name
+|Description
+
+|`bin`
+|Scripts to start and stop ${branding}
+
+|`data`
+|The working directory of the system – installed bundles and their data
+
+|`data/log/${branding-lowercase}.log`
+|Log file for ${branding}, logging all errors, warnings, and (optionally) debug statements. This log rolls up to 10 times, frequency based on a configurable setting (default=1 MB)
+
+|`data/log/ingest.log`
+|Log file for any ingest errors that occur within ${branding}.
+
+|`data/log/security.log`
+|Log file that records user interactions with the system for auditing purposes.
+
+|`deploy`
+|Hot-deploy directory – KARs and bundles added to this directory will be hot-deployed (Empty upon ${branding} installation)
+
+|`documentation`
+|HTML and PDF copies of ${branding} documentation.
+
+|`etc`
+|Directory monitored for addition/modification/deletion of `.config` configuration files or third party `.cfg` configuration files.
+
+|`etc/failed`
+|If there is a problem with any of the `.config` files, such as bad syntax or missing tokens, they will be moved here.
+
+|`etc/processed`
+|All successfully processed `.config` files will be moved here.
+
+|`etc/templates`
+|Template `.config` files for use in configuring ${branding} sources, settings, etc., by copying to the etc directory.
+
+|`lib`
+|The system's bootstrap libraries. Includes the `${branding-lowercase}-branding.jar` file which is used to brand the system console with the ${branding} logo.
+
+|`licenses`
+|Licensing information related to the system.
+
+|`system`
+|Local bundle repository. Contains all of the JARs required by ${branding}, including third-party JARs.
+
+|===
+
+==== Deployment Guidelines
+
+${branding} relies on the Directory Permissions of the host platform to protect the integrity of the ${branding} during operation.
```

Comment:
The directory permission stuff should be done _after_ things have been unzipped, but _before_ DDF is started for the first time. (Hopefully I'm not contradicting my earlier comments :-/)